packages/amplify_core/lib/src/types/exception/error/configuration_error.dart

```diff
@@ -11,7 +11,7 @@ import 'package:amplify_core/amplify_core.dart';
 /// `Amplify.configure`. If that call succeeds, developers do not need to worry
 /// about future configuration errors.
 /// {@endtemplate}
-class ConfigurationError extends AmplifyError {
+class ConfigurationError extends AmplifyError implements ApiException {
```

Comment:
question: why is this implementing `ApiException`? Isn't it used outside API?

Reply:
Shouldn't implement Exception when extending Error